src/DblParser/Lexer.mll

```diff
@@ -142,6 +142,8 @@ rule token = parse
   | lid_start var_char* as x { tokenize_ident x }
   | uid_start var_char* as x { YaccParser.UID x }
   | '`' lid_start var_char* as x { YaccParser.TLID x }
+  | '\'' (_ as ch) '\''   { YaccParser.CHR ch }
+  | "\'\\" (_ as ch) '\'' { YaccParser.CHR (match ch with 'n' -> '\n' | 't' -> '\t' | '\\' -> '\\' | _ -> failwith "Invalid char escape")  }
```

Comment:
I have three comments regarding this single line:

1. In lexing of string literals we do something for escaping sequences. We should not duplicate the code and share this "something" between characters and strings.
2. Raising errors by `failwith "some string without pointing the position in the source"` is very bad. We should use `Error` module. See the reporting invalid escape codes in string literals.
3. This line is too long. We try to keep the source code in 80 columns.